climate_emotions_map/make_descriptive_plots.py

```diff
@@ -179,28 +199,65 @@ def make_descriptive_plot(
     df[COL_CATEGORY] = pd.Categorical(
         df[COL_CATEGORY], categories=CATEGORY_ORDERS[demographic_variable]
     )
-    df = df.sort_values(COL_CATEGORY)
+    df = df.sort_values(COL_CATEGORY, ascending=not reverse)
```

Comment:
> It looks like currently the categories in the plot are ordered exactly as written - not sure if that's your expected/desired behaviour.

Yes it is! The reason why I did this is because originally the bar plots were vertical, so the order was left -> right. But with horizontal bar plots the bars get added from bottom to top and I wanted them to be top -> bottom. So probably not the best way to do it (also there might be a Plotly setting for flipping the y-axis, haven't checked) but it works 😅